DuckDuckGo/PrivacyProtectionEncryptionDetailController.swift

```diff
@@ -83,7 +83,7 @@ class PrivacyProtectionEncryptionDetailController: UIViewController {
 
     private func beginCertificateInfoExtraction() {
         guard siteRating.https else { return }
-        guard let serverTrust = serverTrustCache.get(forDomain: siteRating.domain) else {
+        guard let serverTrust = serverTrustCache.get(forDomain: siteRating.url.host ?? "") else {
```

Comment:
Nice! Optional refactor: Delete the domain property from the SiteRating entirely.

Reply:
I def need to come back to the site rating thing, so will consider that then. Thanks.